src/viv-viewer.js

```diff
@@ -1,29 +1,17 @@
 import React, { PureComponent } from 'react';
 import DeckGL from '@deck.gl/react';
 import { OrthographicView } from '@deck.gl/core';
-import { MicroscopyViewerLayer } from './layers';
-
-export class MicroscopyViewer extends PureComponent {
-  constructor(props) {
-    super(props);
-    this._onWebGLInitialized = this._onWebGLInitialized.bind(this);
-    this.state = {
-      gl: null
-    };
-  }
+import { VivViewerLayer } from './layers';
 
+export class VivViewer extends PureComponent {
   _renderLayers() {
     const { useTiff } = this.props;
-    return new MicroscopyViewerLayer({
-      id: `MicroscopyViewerLayer-${useTiff ? 'tiff' : 'zarr'}`,
+    return new VivViewerLayer({
+      id: `VivViewerLayer-${useTiff ? 'tiff' : 'zarr'}`,
```

Comment:
In other places we're no longer assuming not tiff implies zarr.

Reply:
I'll clarify this.